# Define the goal
float32 forward_velocity  # For manual driving
float32 angular_velocity  # For manual driving

geometry_msgs/PointStamped point # Will not contain orientation information

geometry_msgs/PoseStamped pose   # Will contain orientation information
---
# Define the result
uint32 result
---
# Define a feedback message
float32 percent_complete